tests/general/CLOCBearTest.py

```diff
@@ -0,0 +1,94 @@
+import os
+from queue import Queue
+
+from coalib.settings.Section import Section
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.results.Result import Result
+
```

Comment:
That is two lines after the imports, before the `def`.  It is a PEP8 rule.

But a blank line between `coalib` and `bears` imports is OK.